pycheribuild/projects/cross/ros2.py

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright (c) 2020 Michael Dodson
+# All rights reserved.
+#
+# This software was developed by SRI International and the University of
+# Cambridge Computer Laboratory under DARPA/AFRL contract FA8750-10-C-0237
```

Comment:
I would drop those lines since you weren't funded by DARPA for this change

Reply:
Ok.  It's not yet clear to me what I need to carry over from these statements when I make minor changes to something within the project.